tools/wt_verify.py

```diff
@@ -0,0 +1,156 @@
+# OTHER DEALINGS IN THE SOFTWARE.
+
+# A python script to run the WT tool verify command.
+# This script wraps the wt util tool and takes in arguments for the tool,
+# handles the output and provides additional processing options on the output
+# such as pretty printed output and visualisation options.
+
+# This script only supports Row store and Variable Length Column Store (VLCS).
+# Fixed Length Column Store (FLCS) is not supported.
+
+import argparse
+import os
+import subprocess
+import sys
+
+def visualize(data, visualization_type):
+    """
+    Visualizing data
+    """
+    pass
+
+
+def parse_output(output):
+    """
+    Parse the output file
+    """
+    return output
+
+
+def execute_command(command):
+    """
+    Execute a given command and return its output.
+    """
+    try:
+        result = subprocess.run(command, shell=True, check=True,
+                                stdout=subprocess.PIPE,
+                                universal_newlines=True)
+    except subprocess.CalledProcessError as e:
+        print(f"Error executing command: {e}", file=sys.stderr)
+        sys.exit(1)
+    
+    return result.stdout
+
+
+def find_wt_exec_path():
+    """
+    Find the path of the WT tool executable. 
+    
+    We expect to find exactly one wt binary. Otherwise exit and prompt the user to provide an explicit path to the binary
+    """
+    path = f"{os.path.dirname(os.path.abspath(__file__))}/../."
+
+    try:
+        result = subprocess.run(['find', path, '-maxdepth', '2', '-name', 'wt'],
+                                stdout=subprocess.PIPE, text=True)
+    except subprocess.CalledProcessError as e:
+        print(f"Error executing command: {e}", file=sys.stderr)
+        sys.exit(1)
+
+    list_of_paths = result.stdout.split('\n')
+    # Sanity check to remove empty new lines from find output
+    result = [path for path in list_of_paths if path != '']
+
+    if len(result) > 1:
+        print("Error: multiple wt executables found. Please provide wt executable path using the -wt flag:")
+        for path in result:
+            print(path)
+        exit(1)
+    if len(result) == 0:
+        print("Error: wt executable not found. Please provide path using the -wt flag.")
+        exit(1)
+
+    return result[0]
+
+
+def construct_command(args):
+    """
+    Construct the WiredTiger verify command based on provided arguments.
+    """
+    if args.wt_exec_path:
+        command = f"{args.wt_exec_path} -h {args.home_dir} verify -t"
+    else:
+        command = f"{find_wt_exec_path()} -h {args.home_dir} verify -t"
+
+    if args.dump:
+        command += f" -d {args.dump}"
+    if args.file_name:
+        command += f" \"file:{args.file_name}\""
+    return command
+
+
+def main():
+    parser = argparse.ArgumentParser(description="Script to run the WiredTiger verify command with specified options.")
+    parser.add_argument('-hd', '--home_dir', default='.', help='Path to the WiredTiger database home directory.')
```

Comment:
In the description, we can mention that the current dir is home.

Reply:
ty for the suggestion - done a428a4a0d7c4a13cd0ed3122e64eced8590bb4df